skrifa/src/outline/autohint/shape.rs

```diff
@@ -0,0 +1,87 @@
+//! Shaping support for autohinting.
+
+use raw::{tables::gsub::Gsub, TableProvider};
+
+use crate::{charmap::Charmap, collections::SmallVec, FontRef, GlyphId, MetadataProvider};
+
+/// Determines the fidelity with which we apply shaping in the
+/// autohinter.
```

Comment:
I think I'd expect an enum that determines fidelity to give clearer guidance about the tradeoffs and/or which is better, can we beef up the comment a touch?